src/pyOpenMS/pxds/Matrix.pxd

```diff
@@ -17,22 +17,11 @@ cdef extern from "<OpenMS/DATASTRUCTURES/Matrix.h>" namespace "OpenMS":
         # const_reference getValue(size_t i, size_t j) except + nogil 
         ValueT getValue(size_t i, size_t j) nogil
         void setValue(size_t i, size_t j, ValueT value) nogil
```

Comment:
I removed get and setValue from C++.
How to conveniently wrap this in pyOpenMS?

Reply:
By not removing.
Otherwise you need to change autowrap to allow get_item with tuples.